pom.xml

```diff
@@ -69,7 +69,7 @@ under the License.
         <maven-resources-plugin.version>3.2.0</maven-resources-plugin.version>
         <git-commit-id-maven-plugin.version>5.0.0</git-commit-id-maven-plugin.version>
 
-        <operator.sdk.version>4.1.0</operator.sdk.version>
+        <operator.sdk.version>4.1.1</operator.sdk.version>
```

Comment:
Just for my own understanding, is this needed for `KUBERNETES_NAMESPACE_SYSTEM_PROPERTY`?

Reply:
No this is just so we have the latest josdk version and some fixes